.github/workflows/deploy-to-pages.yaml

```diff
@@ -7,7 +7,7 @@ jobs:
   build-and-test:
     uses: ./.github/workflows/build-and-test.yaml
     with:
-      upload-pages-artifact: true
```

Comment:
Its unfortunate that we missed this the first time around, but I'll try and be more careful about little typos like this in future reviews.